pkg/draft/pack/pack.go

```diff
@@ -34,6 +34,11 @@ const (
 	// ChartsDir is the directory name for the packaged chart.
 	// This also doubles as the directory name for chart dependencies.
 	ChartsDir = "charts"
+	//TasksFileName is the name of the tasks file in a draft pack
+	TasksFileName = "tasks.toml"
+	//TargetTasksFileName is the name of the file where the tasks file from the
+	//  draft pack will be copied to
+	TargetTasksFileName = ".draft-tasks.toml"
```

Comment:
We probably want to reference this in DEP 003, pointing to DEP 008 for further information on Draft Tasks

Reply:
done :)